hudi-common/src/main/java/org/apache/hudi/common/model/HoodieFileGroup.java

```diff
@@ -119,6 +121,10 @@ public void addLogFile(CompletionTimeQueryView completionTimeQueryView, HoodieLo
   @VisibleForTesting
   public String getBaseInstantTime(CompletionTimeQueryView completionTimeQueryView, HoodieLogFile logFile) {
     if (fileSlices.isEmpty()) {
+      if (completionTimeQueryView.getTableVersion().greaterThanOrEquals(HoodieTableVersion.EIGHT)) {
+        // no base file in the file group, use INITIAL INSTANT TIME as base instant time
+        return HoodieActiveTimeline.INIT_INSTANT_TS;
```

Comment:
Looks like a version compatibility issue?

Reply:
> Looks like a version compatibility issue?

Indeed, it is. Essentially, it is because the log file name of the old version contained the base instant time of this slice, while the log file name of the new version only contains the instant written to this file.

Reply:
update: 
When only the log with the smallest instant has not been completed yet, we adopt `INIT_INSTANT_TS` as the base instant because at this time we have no appropriate way to select a reasonable base instant.